compiler/damlc/lib/DA/Cli/Options.hs

```diff
@@ -156,7 +156,7 @@ initPkgDbOpt = InitPkgDb <$> flagYesNoAuto "init-package-db" True "Initialize pa
 
 newtype EnableMultiPackage = EnableMultiPackage {getEnableMultiPackage :: Bool}
 enableMultiPackageOpt :: Parser EnableMultiPackage
-enableMultiPackageOpt = EnableMultiPackage <$> flagYesNoAuto "enable-multi-package" False "Experimental multi-package.yaml support" internal
+enableMultiPackageOpt = EnableMultiPackage <$> flagYesNoAuto "enable-multi-package" True "Enable/disable multi-package.yaml support" internal
```

Comment:
Should this still be internal?

Reply:
Fixed - there were four flags in total that needed to be made visible in help:

For `daml build`: `enable-multi-package`, `all`, `no-cache`
For `daml clean`: `all`